src/OnlineRetailPortal.Core/Models/Product.cs

```diff
@@ -1,25 +1,70 @@
-﻿using System;
+﻿using OnlineRetailPortal.Contracts;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
-namespace OnlineRetailPortal.Core.Models
+using System.Threading.Tasks;
+
+namespace OnlineRetailPortal.Core
 {
     public class Product
     {
         public string Id { get; set; }
         public string Name { get; set; }
-        public string Description { get; set; }
-        public Image HeroImage { get; set; }
         public Price Price { get; set; }
+        public Price IsNegotiable { get; set; }
```

Comment:
IsNegotiable should not be of type Price. verify this

Reply:
done